### ISSUE TITLE
Treviews are no longer read correctly by screen reader

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#13922]
Type: <b>Bug</b>
-isidorn 
When I navigate a treview, screen reader reads all the items at first. When I do that for quite a long time (minute for example) it stops reading those items, though cursor still continues to move across items. In other words, items are still navigable with arrow keys, but screen reader doesn't read them anymore.
You can try this wit a debugger for example. Turn on NVDA screen reader, Write some code that generates an array of items and add a breakpoint in the code after array of items are generated. Then start debugging and go to that breakpoint. Then navigate to debug variables treview and expand that array of items, move arrows up/down for a minute to walk through them and you'll see that items aren't read by NVDA. It happens almost in every treview, not specifically with debugger. It wasn't happening with the previous versions.
I attach a screenshot so that issue is easier understood.
<img width="960" alt="treview_bug" src="https://user-images.githubusercontent.com/34254634/208293242-276c91ea-b53a-4701-b288-b69f817ab5c3.png">

VS Code version: Code 1.74.1 (1ad8d514439d5077d2b0b7ee64d2ce82a9308e5a, 2022-12-14T10:30:51.966Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No
Remote OS version: Linux x64 5.15.0-53-generic
